packages/@tinacms/toolkit/src/packages/fields/plugins/MdxFieldPlugin/updated/plate/plugins/core/autoformat/autoformatBlocks.ts

```diff
@@ -21,6 +21,7 @@ import {
   ELEMENT_CODE_BLOCK,
 } from '@udecode/plate-code-block'
 import { ELEMENT_HR } from '@udecode/plate-horizontal-rule'
+import { ELEMENT_BR } from '@udecode/plate-break'
```

Comment:
I was also getting a type error here as well. Using `"br"` directly seemed to have better luck for me (although might not be totally correct).